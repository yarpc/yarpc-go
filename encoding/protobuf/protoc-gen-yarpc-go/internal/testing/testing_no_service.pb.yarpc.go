// Code generated by protoc-gen-yarpc-go
// source: encoding/protobuf/protoc-gen-yarpc-go/internal/testing/testing_no_service.proto
// DO NOT EDIT!

// Copyright (c) 2019 Uber Technologies, Inc.
//
// Permission is hereby granted, free of charge, to any person obtaining a copy
// of this software and associated documentation files (the "Software"), to deal
// in the Software without restriction, including without limitation the rights
// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
// copies of the Software, and to permit persons to whom the Software is
// furnished to do so, subject to the following conditions:
//
// The above copyright notice and this permission notice shall be included in
// all copies or substantial portions of the Software.
//
// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
// THE SOFTWARE.

package testing

var yarpcFileDescriptorClosurec5c96b9530ffbf1e = [][]byte{
	// encoding/protobuf/protoc-gen-yarpc-go/internal/testing/testing_no_service.proto
	[]byte{
		0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0xe2, 0xf2, 0x4f, 0xcd, 0x4b, 0xce,
		0x4f, 0xc9, 0xcc, 0x4b, 0xd7, 0x2f, 0x28, 0xca, 0x2f, 0xc9, 0x4f, 0x2a, 0x4d, 0x83, 0x30, 0x92,
		0x75, 0xd3, 0x53, 0xf3, 0x74, 0x2b, 0x13, 0x8b, 0x0a, 0x92, 0x75, 0xd3, 0xf3, 0xf5, 0x33, 0xf3,
		0x4a, 0x52, 0x8b, 0xf2, 0x12, 0x73, 0xf4, 0x4b, 0x52, 0x8b, 0x4b, 0x40, 0xaa, 0xa1, 0x74, 0x7c,
		0x5e, 0x7e, 0x7c, 0x71, 0x6a, 0x51, 0x59, 0x66, 0x72, 0xaa, 0x1e, 0x58, 0x9f, 0x90, 0x5d, 0x69,
		0x52, 0x6a, 0x91, 0x1e, 0x58, 0xa3, 0x1e, 0xcc, 0x6c, 0x3d, 0x98, 0xd9, 0x10, 0x46, 0x72, 0x7a,
		0x6a, 0x1e, 0x58, 0x41, 0x7a, 0xbe, 0x1e, 0xcc, 0x60, 0x3d, 0xa8, 0x81, 0x4a, 0xa2, 0x5c, 0xcc,
		0x6e, 0xf9, 0xf9, 0x42, 0x7c, 0x5c, 0x4c, 0x99, 0x29, 0x12, 0x8c, 0x0a, 0x8c, 0x1a, 0x9c, 0x41,
		0x4c, 0x99, 0x29, 0x4e, 0x9c, 0x51, 0xec, 0x50, 0x15, 0x49, 0x6c, 0x60, 0x43, 0x8c, 0x01, 0x01,
		0x00, 0x00, 0xff, 0xff, 0x5a, 0xfb, 0xd2, 0x62, 0xbb, 0x00, 0x00, 0x00,
	},
}
