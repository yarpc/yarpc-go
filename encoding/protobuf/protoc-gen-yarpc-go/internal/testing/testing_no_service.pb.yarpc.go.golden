// Code generated by protoc-gen-yarpc-go
// source: encoding/protobuf/protoc-gen-yarpc-go/internal/testing/testing_no_service.proto
// DO NOT EDIT!

package testing
